assembl/static2/flow/object_types.js

```diff
@@ -1,6 +1,7 @@
 /* @flow */
 /* eslint-disable */
 import { type EditorState } from 'draft-js';
+import { type moment } from 'moment';
```

Comment:
Where did you find there is a type moment to import? There is not moment type in the moment package.